frontend/src/components/dockerDesktopSetup/index.tsx

```diff
@@ -0,0 +1,134 @@
+import { Icon } from '@iconify/react';
```

Comment:
The folder name should match the component name. It's best practice to name the filename also after the component too... and just rexport it from the index.

Can you please add a DockerDesktopSetup.stories.tsx?